pkg/application-grid-controller/controller/common/labels.go

```diff
@@ -31,6 +31,8 @@ const (
 	GridSelectorName = "superedge.io/grid-selector"
 	// GridSelectorKey corresponds to gridUniqKey of the custom-defined workload
 	GridSelectorUniqKeyName = "superedge.io/grid-uniq-key"
+	// TemplateHashKey is a key for storing template's hash value in labels.
+	TemplateHashKey = "service_group_template_hash_key"
```

Comment:
please rename “ service_group_template_hash_key”，keep the same format with GridSelectorName and GridSelectorUniqKeyName